azerothcore/azerothcore-wotlk
# fix(DB/Quest): Very tasty!

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7690
- Closes https://github.com/chromiecraft/chromiecraft/issues/1604

## SOURCE:


## Tests Performed:

- Tested
- Arch


## How to Test the Changes:


1.Tele to NPC 17101 Accept quest
2.go over to node, use item 23654
3.if a murloc doesn't spawn, notice effect.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
